### DISCUSSION TITLE
nfc read with non NDEF

### DISCUSSION THREAD

**Discussion by kmvickers** [shinyorg/shiny#838] [Q&A] [answered]
I have a project in which I have to read an NFC tag which is not formatted NDEF .. is it possible to use shiny nfc to get hold of the raw data from the tag ?

**Comment by aritchie** [comment, marked as answer]
I'm actually not sure.  I doubt it since all of the parsing is based on NDEF.  There is a sample app you can try at https://samples.shinylib.net

**Reply by kmvickers** [reply]
you're right .. its not :) I shall have to go native 

thanks
